examples/quickstart/tutorial/hadoop/docker/yarn-site.xml

```diff
@@ -1,3 +1,21 @@
+<!--
```

Comment:
This (and the other Hadoop XMLs) are probably derivative works of the examples provided by Apache Hadoop. However, since it's another ASF project I think we are okay here, given the policy on https://www.apache.org/legal/src-headers.html#headers.